spec/acceptance/class_disabled_spec.rb

```diff
@@ -45,14 +45,18 @@ class { 'selinux': mode => 'permissive' }
   end
 
   context 'when switching from enforcing to disabled' do
-    let(:pp) do
+    let(:manifest) do
       <<-EOS
         class { 'selinux': mode => 'disabled' }
       EOS
     end
 
     context 'before reboot' do
       before(:all) do
+        # Tag vagrant user to an administrative group
+        shell('semanage login -a -s staff_u vagrant')
```

Comment:
I don't know enough about it. I think the vagrant user is default for vagrant boxes?

Reply:
There's `host['user']` so I think this would ensure the correct user is used:
```suggestion
        block_on hosts do |host|
          on host, "semanage login -a -s staff_u #{host['user']}"
        end
```

I get the impression the rest is all single node, so this is probably the equivalent
```suggestion
        shell("semanage login -a -s staff_u #{default['user']}")
```

Reply:
The vagrant user should be in every vagrant box published, [vagrant docs](https://developer.hashicorp.com/vagrant/docs/boxes/base#default-user-settings). I am happy either way, if a more general fix works we can do that, or any box published should have that user.